profiling/benchmark.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Benchmark context command
+make test > /dev/null
```

Comment:
Aren't we missing a makefile here?

Reply:
no, make has predefined rules for c.
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_10.html

Reply:
Oh right, I wasn't even in `profiling` directory. Nice.